### DISCUSSION TITLE
Manual skip for sponsorblock

### DISCUSSION THREAD

**Discussion by KaziFahmi** [FreeTubeApp/FreeTube#4492] [Q&A] [answered]
Sponsorblock only allows us to auto skip, show in seek and do nothing. In sponsorblock's chrome extension, we can also choose to skip sponsors manually. Is there any way to enable that option?

**Comment by absidue** [comment, marked as answer]
That feature isn't implemented yet, so no there isn't currently an option you can enable it.

**Reply by KaziFahmi** [reply]
Do you know if there are plans on implementing that feature?

**Reply by absidue** [reply]
There is an open feature request for it here: https://github.com/FreeTubeApp/FreeTube/issues/1380
